shema_based_on_diagram.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE patients (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name VARCHAR(100),
+  date_of_birth DATE
+);
+CREATE TABLE medical_histories (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id BIGINT NOT NULL REFERENCES patients(id),
+  status VARCHAR(100)
+);
+
+CREATE TABLE invoices (
+  id BIGSERIAL NOT NULL  PRIMARY KEY,
+  total_amount DECIMAL,
+  admitted_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_histories_id BIGINT NOT NULL UNIQUE REFERENCES medical_histories
+);
```

Comment:
- [ ] Please change the `admitted_at` column with `generated_at` to follow the diagram schema.

![image](https://user-images.githubusercontent.com/83803414/181571896-8629346e-b58b-4819-a3ff-7a850069ad13.png)